test_util/helpers.py

```diff
@@ -173,3 +175,10 @@ def remove_file():
     # Attempt to remove the file upon normal interpreter exit.
     atexit.register(remove_file)
     return temp_path
+
+
+def parse_env_file(path):
+    """ converts any file with one key=value assignment per line
+    into a python dict
+    """
+    return {i[0]: '='.join(i[1:]) for i in [j.split('=') for j in load_string(path).split('\n')]}
```

Comment:
there are comments in the file, and the splits should be limited. If we want ot write this function, will also need to write a unit test for it, I can give some cases which will break this which are fairly common.